tss-esapi/src/structures/buffers/sensitive_create.rs

```diff
@@ -0,0 +1,174 @@
+// Copyright 2022 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+use crate::{
+    structures::SensitiveCreate,
+    traits::{Marshall, UnMarshall},
+    tss2_esys::{TPM2B_SENSITIVE_CREATE, TPMS_SENSITIVE_CREATE},
+    Error, Result, ReturnCode, WrapperErrorKind,
+};
+use log::error;
+use std::{
+    convert::{TryFrom, TryInto},
+    ops::Deref,
+};
+use zeroize::Zeroize;
+/// Sensitive create buffer
```

Comment:
Maybe a newline before the doc comment would look nice? (also for the other struct).

Reply:
Fixed